packages/database/src/schema.types.ts

```diff
@@ -186,6 +186,15 @@ export type Withdrawal = {
   proposal?: undefined | null
 }
 
+export type InstantWithdrawal = {
+  signature: string
+  withdrawalHash: string
+  prepayFeeInEth: string
+  prepayFeeInToken: string
+  expiration: undefined
```

Comment:
I compiled well with `expiration: number` instead of `undefined`, is possible no expiration?

Reply:
Yes, it should be `expiration: number`, I re-ran the type generator script.